frontend/src/components/SetupWizard.vue

```diff
@@ -118,6 +129,33 @@ export default {
         this.model.installationFolder.startsWith("/")
       );
     },
+
+    validateConfigFolder() {
```

Comment:
there's also a method in `compute`, and the method name doesn't match what the method is actually doing